Tests/BackendServiceTests.swift

```diff
@@ -1,179 +0,0 @@
-////
-////  BackendServiceTests.swift
-////  Tests
-//
-////  Copyright © 2017 Bottle Rocket Studios. All rights reserved.
-////
-//
-//import XCTest
-//@testable import Hyperspace
-//
-//class BackendServiceTests: XCTestCase {
```

Comment:
These are just tests I had commented out so I could get them working again piece-meal, rebuilding these tests in the new `async` architecture is an outstanding task.

Reply:
Alright, got the tests back in and working (at least passing on my machine!)